tests/acceptance/blockchain/agents/test_staking_escrow_agent.py

```diff
@@ -264,16 +263,6 @@ def test_deposit_and_increase(agency, testerchain, test_registry, token_economic
     assert staking_agent.get_locked_tokens(staker_account, 1) == locked_tokens + amount
 
 
-def test_disable_restaking(agency, testerchain, test_registry):
```

Comment:
Did this test get relocated somewhere?

Reply:
Few lines below - same method with the same name and this one was already disabled